webview/jest.config.js

```diff
@@ -17,8 +17,10 @@ module.exports = {
   moduleNameMapper: {
     '\\.(scss|css|less)$': 'identity-obj-proxy'
   },
-  preset: 'ts-jest',
   setupFiles: ['jest-canvas-mock', '<rootDir>/setup-tests.js'],
-  testEnvironment: 'node',
-  testTimeout: 20000
+  testEnvironment: 'jsdom',
```

Comment:
[F] Has to be `jsdom` or everything blows up. Which is fine, we were only doing this to try and get some performance improvements.